websites/L/Leboncoin/presence.ts

```diff
@@ -0,0 +1,110 @@
+		} else if (document.location.pathname.includes("/profil/")) {
+			presenceData.details = "Regarde la page :";
+			const profilename = document.querySelector(
+				"#mainContent > div.styles_Profile__3vhTo > div.styles_ProfileHero__aQISv > div > div.styles_userInfo__XBcBm > div > h3"
+			).textContent;
+			presenceData.state = `Profil de ${profilename}`;
+			presenceData.buttons = [
+				{ label: "Consulter le profil", url: `${document.URL}` }
+			];
+		} else if (document.location.pathname.includes("/blog")) {
+			presenceData.details = "Viewing Page:";
+			presenceData.state = "Node.js News";
+		} else if (document.location.pathname.includes("/recherche")) {
+			presenceData.details = "Dans les résultats de recherche :";
+
+			presenceData.state = `Annonces pour « ${
+				document
+					.querySelectorAll("h1")[0]
+					.innerHTML.split("«")
+					.pop()
+					.split("»")[0]
+			} »`;
+		} else if (document.location.pathname.includes("/commerces_marches/")) {
+			presenceData.details = `Annonce ${document.title.split("-")[0]}`;
+			const nomannonceur = document.querySelector<HTMLSpanElement>(
+				"#aside > section > div > div.styles_ProfileLight__esTKc > div.styles_ProfileLightContent__Jg3FU > div.styles_content__XvyPv > div.styles_heading__1Vz_9 > a"
+			).textContent;
+			presenceData.state = `Vendu par ${nomannonceur}`;
+			presenceData.buttons = [
+				{ label: "Consulter l'annonce", url: `${document.URL}` }
+			];
+		} else if (document.location.pathname.includes("/voitures/")) {
+			presenceData.details = `Annonce ${document.title.split("-")[0]}`;
+			const nomannonceur = document.querySelector<HTMLSpanElement>(
+				"#aside > section > div > div.styles_ProfileLight__esTKc > div.styles_ProfileLightContent__Jg3FU > div.styles_content__XvyPv > div.styles_heading__1Vz_9 > a"
+			).textContent;
+			presenceData.state = `Vendu par ${nomannonceur}`;
+			presenceData.buttons = [
+				{ label: "Consulter l'annonce", url: `${document.URL}` }
+			];
+		} else if (document.location.pathname.includes("/")) {
+			presenceData.details = `Annonce ${document.title.split("-")[0]}`;
+			const prixannonce = document.querySelector(
+					"#mainContent > div > div > nav > div._3IX1b > div > div > div.sc-bdVaJa.cJzirk > div.sc-bdVaJa.gleyua > div.sc-bdVaJa.gKAIQK > div > span"
+				).textContent,
+				nomannonceur = document.querySelector<HTMLSpanElement>(
+					"#aside > section > div > div.styles_ProfileLight__esTKc > div.styles_ProfileLightContent__Jg3FU > div.styles_content__XvyPv > div > a._3k87M._3Hrjq._3Wx6b._2MFch._1hnil._35DXM._1-TTU._1GcfX._2DyF8._3k00F"
+				).textContent;
+			presenceData.state = `Vendu ${prixannonce} par ${nomannonceur}`;
+			presenceData.buttons = [
+				{ label: "Consulter l'annonce", url: `${document.URL}` }
+			];
+		} else if (document.location.pathname === "https://www.leboncoin.fr") {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Accueil du site";
+		}
```

Comment:
you should add an else here to set details if no conditions match and remove the defaultPresenceData variable